.github/workflows/build_and_push_nonproduction_images.yml

```
@@ -10,90 +10,193 @@ on:
 
 env:
   REGISTRY: ghcr.io
+  ORG_NAME: refactor-group
+  REPO_NAME: refactor-platform-fe
 
 jobs:
-  build_and_push_frontend:
+  build_and_push_amd64:
     runs-on: ubuntu-22.04
```

```suggestion
    runs-on: ubuntu-24.04
```
To be consistent with the backend builder version.